sandbox/interface_linux.go

```diff
@@ -311,3 +340,25 @@ func setInterfaceRoutes(iface netlink.Link, i *nwIface) error {
 	}
 	return nil
 }
+
+const (
+	netStatsFile = "/proc/net/dev"
+	base         = "[ ]*%s:([ ]+[0-9]+){16}"
+)
+
+func scanInterfaceStats(data, ifName string, i *InterfaceStatistics) error {
+	var (
+		bktStr string
+		bkt    uint64
+	)
+
+	regex := fmt.Sprintf(base, ifName)
+	re := regexp.MustCompile(regex)
+	line := re.FindString(data)
+
+	_, err := fmt.Sscanf(line, "%s %d %d %d %d %d %d %d %d %d %d %d %d %d %d %d %d",
+		&bktStr, &i.RxBytes, &i.RxPackets, &i.RxErrors, &i.RxDropped, &bkt, &bkt, &bkt,
+		&bkt, &i.TxBytes, &i.TxPackets, &i.TxErrors, &i.TxDropped, &bkt, &bkt, &bkt, &bkt)
+
```

Comment:
Not too comfortable with parsing the formatted file to get to the desired data. Seems very fragile. Libcontainer used another approach which seems more appropriate :
` // Reads the specified statistics available under /sys/class/net/<EthInterface>/statistics`